### PR TITLE
docs(toh-5): copyedits

### PR DESCRIPTION
**Pull request by chalin**

### CODE REVIEW

**Comment by chalin** [comment]
@kapunahelewong - pls review & approve/comment when you have a minute. Thanks!

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 1]
```diff
@@ -5,7 +5,7 @@ block includes
   - var _appRoutingTsVsAppComp = 'app.module.ts'
   - var _RoutesVsAtRouteConfig = 'Routes'
   - var _RouterModuleVsRouterDirectives = 'RouterModule'
-  - var _redirectTo = 'redirectTo'
+  - var _redirect = 'redirect'
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Why is this here?

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 2]
```diff
@@ -65,13 +65,14 @@ block keep-app-running
     ## Keep the app transpiling and running
     Enter the following command in the terminal window:
 
-code-example(language="sh" class="code-shell").
-  npm start
+  code-example(language="sh" class="code-shell").
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: This indentation seems wrong. The original seemed correct. What am I missing?

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 3]
```diff
@@ -307,11 +308,11 @@ block routerLink
   +makeExcerpt('src/app/app.module.ts', 'dashboard')
 
 :marked
-  ### Add a redirect route
+  ### Add a !{_redirect} route
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Why `!{_redirect}` ?  We're getting rid of these.

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 4]
```diff
@@ -307,11 +308,11 @@ block routerLink
...
 
+  Currently, the browser launches with `/` in the address bar.
   When the app starts, it should show the dashboard and
   display a `/dashboard` URL in the browser address bar.
-  Currently, the browser launches with `/` in the address bar.
 
 block redirect-vs-use-as-default
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Why is this here? Remove and slide the following jade to the left.

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 5]
```diff
@@ -504,7 +503,7 @@ block route-params
 
 block ngOnInit
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Remove and adjust left

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 6]
```diff
@@ -519,7 +518,7 @@ block extract-id
     If a user re-navigates to this component while a `getHero` request is still processing,
     `switchMap` cancels the old request and then calls `HeroService.getHero()` again.
 
-- var _str2int = _docsFor == 'dart' ? '<code>int.parse</code> static method' : 'JavaScript (+) operator'
+- var _str2int = _docsFor == 'dart' ? '<code>int.parse()</code> static method' : 'JavaScript (+) operator'
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Remove variable. Use text.

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 7]
```diff
@@ -554,13 +553,12 @@ block extract-id
 
 +makeExcerpt('src/app/hero-detail.component.ts', 'goBack')
 
-- var _CanDeactivateGuard = _docsFor == 'dart' ? '<em>routerCanDeactivate</em> hook' : '<em>CanDeactivate</em> guard'
-- var _CanDeactivateGuardUri = _docsFor == 'dart' ? 'angular2.router/CanDeactivate-class' : 'router/index/CanDeactivate-interface'
+- var _CanDeactivateGuard = _docsFor == 'dart' ? '<em>routerCanDeactivate()</em> hook' : '<em>CanDeactivate</em> guard'
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Remove variable.

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 8]
```diff
@@ -660,14 +658,13 @@ block extract-id
     Delete the routing configuration from `AppModule` and import the `AppRoutingModule`.
     Use an ES `import` statement *and* add it to the `NgModule.imports` list.
 
-:marked
     Here is the revised `AppModule`, compared to its pre-refactor state:
 
-+makeTabs(
+  +makeTabs(
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Why indented?

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 9]
```diff
@@ -771,14 +768,14 @@ block heroes-component-cleanup
...
-  1. Implement `gotoDetail()` by calling the `router.navigate()` method.
+  1. Implement `gotoDetail()` by calling the router `navigate()` method.
 
 +makeExcerpt('src/app/heroes.component.ts', 'gotoDetail')
 
 :marked
   Note that you're passing a two-element *link parameters !{_array}*&mdash;a
-  path and the route parameter&mdash;to
-  the `router.navigate()` method, just as you did in the `[routerLink]` binding
+  !{_pathVsName} and the route parameter&mdash;to
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Remove variables. Replace all `!{_array}` with `array`

File: public/docs/ts/latest/tutorial/toh-pt5.jade (hunk 1 above)
Comment (chalin) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: It was here from before, but just not updated. This is fixing it.

File: public/docs/ts/latest/tutorial/toh-pt5.jade (hunk 2 above)
Comment (chalin) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: The code sample is part of the preceding block.

File: public/docs/ts/latest/tutorial/toh-pt5.jade (hunk 3 above)
Comment (chalin) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Yes, in the next pass ... :)

File: public/docs/ts/latest/tutorial/toh-pt5.jade (hunk 2 above)
Comment (Foxandxss) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: And Ward was right. It was wrong. I need to revert this
